scripts/Github_project_to_CSV.py

```diff
@@ -0,0 +1,124 @@
+"""
+poetry init -n
+poetry add requests
+poetry add pprintpp
+poetry run python
+"""
+
+import json
+import csv
+
+import requests
+from pprintpp import pprint as pp
+
+# https://github.com/settings/tokens/819269469
+token = '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'
```

Comment:
Remove token and move it as an command argument. In order to do that, you need to move everything under `main` function, you can see how this can be done by using this example:

https://github.com/atviriduomenys/katalogas/blob/devel/scripts/anonymize.py